### ISSUE TITLE
How add Key-value pairs to associate with  this  stack?

### ISSUE THREAD

**Issue by opsaaspo** [tongueroo/lono#39]
Hello,

How can I add key-value pairs to associate with  this  stack when creating or updating or importing a stack?
For example:
```
aws cloudformation create-stack --tags "<some keys and values>" ...
aws cloudformation update-stack --tags "<some keys and values>" ...
```

**Comment by tongueroo** [comment]
Interesting, `lono cfn create` and `lono cfn update` currently doesn't support tagging the stack. Looks like just have to pass the options down from the CLI to the aws-sdk call: https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/CloudFormation/Client.html#create_stack-instance_method Unsure when will add this. Happy to consider pull requests for this. 👌

**Comment by tongueroo** [comment]
@opsaaspo Released in [v4.2.7](https://github.com/tongueroo/lono/blob/master/CHANGELOG.md)

Example:

    lono cfn create example --tags a:1 b:2
    lono cfn update example # retains current values
    lono cfn update example --tags c:3 # overwrites current values
